### ISSUE TITLE
Unable to connect; error during the init..

### ISSUE THREAD

**Issue by rclvb** [AZn5ReD/saplearninghub-pdf-downloader#16]
I am unable to connect using the new version..see screenshot
![image](https://user-images.githubusercontent.com/77281601/104852956-aaacf880-58cb-11eb-8950-b750f522f8db.png)

**Comment by sag8** [comment]
I got this as well. This is because a new popup has been added by SAP for accepting new terms of use. So, the code doesn't handle that automatically.

All I did was:
1. Opened chrome and just logged into this address - [https://performancemanager.successfactors.eu/sf/learning?company=learninghub](url) using the credentials I used in config.json. 

2. Once logged in, clicked on the "Accept" button to accept the new terms and closed any other dialog box that follows. 

That's it. Try again and the download should work.
